hw/ip/otbn/dv/otbnsim/BUILD

```diff
@@ -0,0 +1,11 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@rules_python//python:defs.bzl", "py_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+# Make standalone simulator visible to OTBN tests within Bazel (see
+# rules/otbn.bzl).
+exports_files(["standalone.py"])
```

Comment:
I think, adding this BUILD file is going to hide files in hw/ip/otbn/dv/otbnsim from the hw:all_files group, which is probably fine because it looks self contained so that this default behavior is what we want. If we want to keep the files in hw:all_files we'd want to add an all_files glob here and add it in to the BUILD file here https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/otbn/BUILD

Reply:
We should also transform the simulator, its libraries and its tests into a proper executable with `py_{binary, library, test}` rules.

Reply:
I think this one will be a fairly large change -- is it okay if I make an issue and do it as a follow-up? I've added an extra note in the `otbnsim/BUILD` file to clarify the `all_files` consequences, and a TODO in `otbn.bzl` that says the simulator should be made a Bazel executable.

Reply:
I think it's fine, it might lead us to assign a P1 around when we start working on reducing dependencies in the getting started page or if it collides with an incompatible CI commit when it gets merged in. Until then I'd rather have the test enforced in CI than not have it at all.